flink-cdc-runtime/src/main/java/org/apache/flink/cdc/runtime/functions/SystemFunctionUtils.java

```diff
@@ -645,4 +645,32 @@ private static String castObjectIntoString(Object object) {
         }
         return String.valueOf(object);
     }
+
+    public static boolean greater(Comparable comparable1, Comparable comparable2) {
```

Comment:
To keep consistency with [Flink](https://github.com/apache/flink/blob/ed50ae21fc78fa98b015566303f9a80c88278df1/flink-table/flink-table-common/src/main/java/org/apache/flink/table/functions/BuiltInFunctionDefinitions.java#L579) and function names, maybe we can use `greaterThan`, `greaterThanOrEqual`, `lessThan`, `lessThanOrEqual`?

Reply:
I agree with you.